### DISCUSSION TITLE
How can you type a component and use `props.children`?

### DISCUSSION THREAD

**Discussion by brandonpittman** [BuilderIO/jsx-lite#1621] [Q&A]
I can't figure out how to type `children` and adding prop types to a component that uses children makes TypeScript angry.

**Comment by samijaber** [comment]
We currently resort to `props.children: any`. The reason for that is this issue: https://github.com/BuilderIO/mitosis/issues/461

we have not yet built a mapping of Mitosis JSX types to each framework's types. This means that annotating anything framework-specific is not yet possible (children, nodes, event handlers, etc.)

I'm going to close this discussion since it is blocked by the issue I linked above.
